usaspending_api/accounts/tests/test_federal_endpoints.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from model_mommy import mommy
+import json
+
+
+@pytest.fixture
+def federal_account_models():
+    fed_1 = mommy.make('accounts.FederalAccount', account_title="Community Spending", _fill_optional=True)
+    fed_2 = mommy.make('accounts.FederalAccount', account_title="Military Spending", _fill_optional=True)
+    fed_3 = mommy.make('accounts.FederalAccount', account_title="Foreign Spending", _fill_optional=True)
+    tas_1 = mommy.make('accounts.TreasuryAppropriationAccount', federal_account=fed_1, tas_rendering_label="ABC", _fill_optional=True)
```

Comment:
I figured eventually, when we add sums on the federal account level by associated tas level data it would be useful. (Also largely leftover from a bit of copy and paste, so I can remove if needbe)